tchannel/request.py

```diff
@@ -47,6 +47,16 @@ class Request(object):
         The most useful piece of information here is probably
         ``request.transport.caller_name``, which is the identity of the
         application that created this request.
+
+    :ivar service:
+        Name of the service being called.
```

Comment:
How would this differ from the name you advertised as? Is this related to rd transport header?